host/class/hid/usb_host_hid/hid_host.c

```diff
@@ -829,7 +829,7 @@ static esp_err_t usb_class_request_get_descriptor(hid_device_t *hid_device, cons
 /**
  * @brief HID Host Request Report Descriptor
  *
- * @param[in] hidh_iface      Pointer to HID Interface configuration structure
+ * @param[in] hid_iface       Pointer to HID Interface configuration structure
```

Comment:
```suggestion
 * @param[in] iface       Pointer to HID Interface configuration structure
```

Reply:
Now fixed, also couple of others like this fixed as well.